src/Sulu/Bundle/MediaBundle/Resources/config/doctrine/FileVersionMeta.orm.xml

```diff
@@ -6,7 +6,6 @@
             table="me_file_version_meta"
             repository-class="Sulu\Bundle\MediaBundle\Entity\FileVersionMetaRepository">
         <indexes>
-            <index columns="title"/>
```

Comment:
I think the title don't need an index because the database can't use an Index with '%value%' ? /cc @sulu/core-team

Reply:
Same as above